apps/etterlatte-behandling/src/main/kotlin/behandling/etteroppgjoer/EtteroppgjoerDao.kt

```diff
@@ -103,6 +102,53 @@ class EtteroppgjoerDao(
             }
         }
 
+    fun hentEtteroppgjoerSakerIBulk(
+        inntektsaar: Int,
+        antall: Int,
+        etteroppgjoerFilter: EtteroppgjoerFilter,
+        spesifikkeSaker: List<SakId>,
+        ekskluderteSaker: List<SakId>,
+    ): List<SakId> =
+        connectionAutoclosing.hentConnection {
+            with(it) {
+                prepareStatement(
+                    """
+                    SELECT sak_id FROM etteroppgjoer e 
+                    WHERE e.status = 'MOTTATT_SKATTEOPPGJOER'
+                    AND e.inntektsaar = ?
+                    AND e.har_sanksjon = ?
+                    AND e.har_institusjonsopphold = ?
+                    AND e.har_opphoer = ?
+                    AND e.har_bosatt_utland = ?
+                    AND e.har_adressebeskyttelse_eller_skjermet = ?
+                    AND e.har_aktivitetskrav = ?
+                     ${if (spesifikkeSaker.isEmpty()) "" else " AND sak_id = ANY(?)"}
+                     ${if (ekskluderteSaker.isEmpty()) "" else " AND sak_id = ANY(?)"}
```

Comment:
sjekken skal være motsatt for ekskluderte saker, ellers så blir de inkludert slik det står nå